SETUP.md

```diff
@@ -1,11 +1,11 @@
 # AutoSpotting Setup #
```

Comment:
I would have renamed this file CUSTOM.md or something similar, as it's more about using custom version of the project, rather than setupping the project.